cli/command/stack/deploy_composefile.go

```diff
@@ -208,6 +216,33 @@ func createSecrets(
 	return nil
 }
 
+func createConfigs(
+	ctx context.Context,
+	dockerCli command.Cli,
+	namespace convert.Namespace,
+	configs []swarm.ConfigSpec,
+) error {
+	client := dockerCli.Client()
+
+	for _, configSpec := range configs {
+		secret, _, err := client.ConfigInspectWithRaw(ctx, configSpec.Name)
+		if err == nil {
+			// secret already exists, then we update that
+			if err := client.ConfigUpdate(ctx, secret.ID, secret.Meta.Version, configSpec); err != nil {
```

Comment:
s/secret/config/ 

The variable above here, the comment, and the comment below this as well